threads-web-ui/.gitignore

```
@@ -25,10 +25,11 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
+.env*.local
 
 # vercel
 .vercel
+
+# typescript
+*.tsbuildinfo
+next-env.d.ts
```

```suggestion
next-env.d.ts
```
Why are you ignoring this file from version control?

---

The `next-env.d.ts` file is not required for version control purposes. It is a TypeScript declaration file specific to Next.js projects. This file is automatically generated by Next.js when you run the development server (next dev) or build your application (next build).
It is recommended that each developer generates their own `next-env.d.ts` file to ensure compatibility with their local development environment and TypeScript setup.

---

I see :eyes: 
Okay! I will remove it from the `.gitignore` file. :smile: